### DISCUSSION TITLE
Can you add "Brave-Fox" theme in floorp browser as optional like many others present?

### DISCUSSION THREAD

**Discussion by ghost** [Floorp-Projects/Floorp#831] [Feature Request]
Can you add "Brave-Fox" theme in floorp browser as optional like many others present?

https://github.com/Soft-Bred/Brave-Fox

**Comment by Telmesomething** [comment]
You can download it from the Mozilla addons site at: https://addons.mozilla.org/en-US/firefox/addon/brave-fox/versions/

**Comment by ghost** [comment]
> You can download it from the Mozilla addons site at: https://addons.mozilla.org/en-US/firefox/addon/brave-fox/versions/

Why older version?
